main/src/org/destinationsol/menu/InputMapScreen.java

```diff
@@ -279,33 +282,30 @@ public boolean reactsToClickOutside() {
     }
 
     @Override
-    public boolean isCursorOnBg(SolInputManager.Ptr ptr) {
+    public boolean isCursorOnBg(SolInputManager.Ptr pointers) {
         return false;
     }
 
     @Override
-    public void onAdd(SolApplication cmp) {
+    public void onAdd(SolApplication solApplication) {
         // Add any extra screen information as required by the input screens. E.g. buttons
         if (operations != null) {
-            cmp.getInputMan().addScreen(cmp, operations);
+            solApplication.getInputMan().addScreen(solApplication, operations);
         }
 
         page = 0;
         selectedIndex = 0;
     }
 
     @Override
-    public void blurCustom(SolApplication cmp) {
+    public void blurCustom(SolApplication cmp) { }
```

Comment:
I seem to recall a recent discussion triggered by FC regarding this, and manu and slardarnar/immotius settled on { } as opposed to { \n }